### PR TITLE
Update to new release guidance

### PR DESCRIPTION
**Pull request by Brunoga-MS**
# Overview/Summary

Adding documentation page to update existing deployments to a new release of AMBA

## This PR fixes/adds/changes/removes

1. docs/content/patterns/alz/Update-to-new-Release.md as part of [AMBA Create update guidance](https://dev.azure.com/CSUSolEng/Azure%20Landing%20Zones/_backlogs/backlog/Azure%20Landing%20Zones%20Team/Stories/?workitem=31687)
2. related images

### Breaking Changes

1. NONE

## As part of this Pull Request I have

- [X] Read the Contribution Guide and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/azure-monitor-baseline-alerts/pulls)
- [X] Associated it with relevant [GitHub Issues](https://github.com/Azure/azure-monitor-baseline-alerts/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/azure-monitor-baseline-alerts/)
- [X] Ensured PR tests are passing
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: patterns/alz/AMBA_Demo-alzArm.param.json
Comment (arjenhuitema) [inline patterns/alz/AMBA_Demo-alzArm.param.json:1]: Please review, looks like a test param file.

**Review by arjenhuitema — CHANGES_REQUESTED** [review]
Thanks Bruno! Please review the 3 additional parameter files. It looks like those were from a demo/ test environment, if so please removed them from the PR.

File: docs/content/patterns/alz/Update-to-new-Release.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    ![Confirm pull](../media/ConfirmPull.png)
+
+4. Ensure there are no issues with pulling changes and that the numbers close to the icon in the bottom-left corner are both **0** or not showing at all
+
+    ![Pull request completed](../media/PullCompleted.png)
+
+</br>
+
+### Deploy
```
Comment (paulgrimley) [inline docs/content/patterns/alz/Update-to-new-Release.md:63]: What about if you deploy using non-GH approach?

**Review by paulgrimley — CHANGES_REQUESTED** [review]
Great addition, just need to add consideration for offline deployment

File: patterns/alz/AMBA_Demo-alzArm.param.json
Comment (Brunoga-MS) [reply patterns/alz/AMBA_Demo-alzArm.param.json:1]: removed

File: docs/content/patterns/alz/Update-to-new-Release.md (hunk 1 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/Update-to-new-Release.md]: done

**Review by paulgrimley — APPROVED** [review]
Thanks for updating, LGTM

**Review by arjenhuitema — APPROVED** [review]
LGTM

**Pull request merged**